code/modules/mob/language/station_vr.dm

```diff
@@ -179,6 +186,10 @@
 /datum/language/echosong/scramble(var/input, var/list/known_languages)
 	return stars(input)
 
```

Comment:
missing comments, why was this upstream file edits just to move some code around?

Reply:
I moved it because it was unorganized.